index.html

```diff
@@ -4391,127 +4391,61 @@ <h2>Choosing DID Resolvers</h2>
-<a>public key description</a> in the <a>DID document</a>: static and dynamic.
-        </p>
-
-        <p>
-The static method is to sign the <a>DID document</a> with the private key. This
-proves control of the private key at a time no later than the
-<a>DID document</a> was registered. If the <a>DID document</a> is not signed,
-control of a public key described in the <a>DID document</a> can still be proven
-dynamically as follows:
-        </p>
-
-        <ol start="1">
-          <li>
-Send a challenge message containing a <a>public key description</a> from the
-<a>DID document</a> and a nonce to an appropriate <a>service endpoint</a>
-described in the <a>DID document</a>.
-          </li>
-
-          <li>
-Verify the signature of the response message against the
-<a>public key description</a>.
-          </li>
-        </ol>
       </section>
 
-      <section>
-        <h3>Real-World Identity</h3>
+      <section class="notoc">
+        <h3>Binding to Physical Identity</h3>
 
         <p>
 A <a>DID</a> and <a>DID document</a> do not inherently carry any
-<a href="https://en.wikipedia.org/wiki/Personal_data">
-personal data</a>.
+<a href="https://en.wikipedia.org/wiki/Personal_data">personal data</a> and
+it is strongly advised that non-public entities do not publish personal data in
+<a>DID documents</a>.
         </p>
 
         <p>
-It can be useful to express a binding of <a>DID</a> to a person's or company's
-real world identity, in a way that is provably asserted by a trusted authority
-such as a government. This can enable interactions that can be considered
-legally valid under one or more jurisdictions; establishing such bindings has to
-be carefully balanced against privacy considerations (see <a
-href="#privacy-considerations"></a>).
+It can be useful to express a binding of a <a>DID</a> to a person's or
+organization's physical identity in a way that is provably asserted by a
+trusted authority, such as a government. This specification provides a
+<a>verification relationship</a> in <a href="#assertion"></a> for these
+purposes. This feature can enable interactions that are private and can be
+considered legally enforceable under one or more jurisdictions; establishing
+such bindings has to be carefully balanced against privacy considerations (see
+<a href="#privacy-considerations"></a>).
         </p>
 
         <p>
 The process of binding a <a>DID</a> to something in the real world, such as a
-person or a company, for example using <a>verifiable credentials</a> with the
-same subject as that <a>DID</a>, is out of scope for this specification. For
-more information, see the Verifiable Credentials Data Model [[VC-DATA-MODEL]].
+person or an organization, for example using <a>verifiable credentials</a> with
+the same subject as that <a>DID</a>, is contemplated by this specification
+and further defined in the Verifiable Credentials
+Data Model [[VC-DATA-MODEL]].
```

Comment:
```suggestion
The process of binding a <a>DID</a> to something in the physical world, such
as a person or an organization &mdash; for example, by using <a>verifiable
credentials</a> with the same subject as that <a>DID</a> &mdash; is
contemplated by this specification and further defined in the Verifiable
Credentials Data Model [[VC-DATA-MODEL]].
```

Reply:
Applied in fe809af3db8f85296af1bcc49648efa470c572a4.